libcxx/include/experimental/__simd/simd.h

```diff
@@ -44,6 +58,10 @@ class simd {
 
   _LIBCPP_HIDE_FROM_ABI simd() noexcept = default;
 
+  // explicit conversion from and to implementation-defined types
+  explicit _LIBCPP_HIDE_FROM_ABI operator _Storage() const;
+  explicit _LIBCPP_HIDE_FROM_ABI simd(const _Storage& __s) : __s_(__s) {}
```

Comment:
I think we should do this with an explicit member and a constructor with a tag type instead. Otherwise people are going to start relying on it. Also, why is the conversion operator not implemented?